### PR TITLE
[CHF-156] Health Check NYCE Door/Window Sensor NCZ 3011

### PR DESCRIPTION
**Pull request by parijatdas**
Added health checks for NYCE door/window sensor
checkInterval value determined and added
Implemented ping functionality

Please check and merge the changes.
@jackchi @ShunmugaSundar

### CODE REVIEW

**Comment by jackchi** [comment]
👍 looks good.

**Comment by parijatdas** [comment]
@jackchi We've removed the temporary fix for ping and changed the checkInterval to 12min with the reporting interval being configured at 5min. Please check and merge the changes.

File: devicetypes/smartthings/nyce-open-closed-sensor.src/nyce-open-closed-sensor.groovy [hunk 1]
```diff
@@ -18,26 +18,27 @@ import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
 
 
 metadata {
-	definition (name: "NYCE Open/Closed Sensor", namespace: "smartthings", author: "NYCE") {
+	definition (name: "NYCE Open/Closed Sensor", namespace: "smartthings", author: "NYCE", category: "C2") {
     	capability "Battery"
 		capability "Configuration"
```
Comment (jackchi) [inline devicetypes/smartthings/nyce-open-closed-sensor.src/nyce-open-closed-sensor.groovy]: Odd formatting here.

File: devicetypes/smartthings/nyce-open-closed-sensor.src/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+
+Works with:
+
+* [Samsung SmartThings Nyce Door/Window Sensor](https://support.smartthings.com/hc/en-us/articles/204576764-NYCE-Door-Window-Sensor)
+
+## Table of contents
+
+* [Capabilities](#capabilities)
+* [Health]($health)
```
Comment (jackchi) [inline devicetypes/smartthings/nyce-open-closed-sensor.src/README.md]: Broken link.

**Review by jackchi — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/nyce-open-closed-sensor.src/nyce-open-closed-sensor.groovy (hunk 1 above)
Comment (parijatdas) [reply devicetypes/smartthings/nyce-open-closed-sensor.src/nyce-open-closed-sensor.groovy]: Fixed the indentation in the metadata section. Please check.

File: devicetypes/smartthings/nyce-open-closed-sensor.src/README.md (hunk 2 above)
Comment (parijatdas) [reply devicetypes/smartthings/nyce-open-closed-sensor.src/README.md]: This is fixed now. Please check.

File: devicetypes/smartthings/nyce-open-closed-sensor.src/nyce-open-closed-sensor.groovy [hunk 3]
```diff
@@ -18,26 +18,27 @@ import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
 
 
 metadata {
-	definition (name: "NYCE Open/Closed Sensor", namespace: "smartthings", author: "NYCE") {
-    	capability "Battery"
+	definition (name: "NYCE Open/Closed Sensor", namespace: "smartthings", author: "NYCE", category: "C2") {
```
Comment (jackchi) [inline devicetypes/smartthings/nyce-open-closed-sensor.src/nyce-open-closed-sensor.groovy]: Let's remove `Category`.

**Review by jackchi — APPROVED** [review]

**Comment by jackchi** [comment]
👍  Looks good once we remove the Category.

**Pull request merged**